locales/en-US/app.l20n

```diff
@@ -95,3 +95,7 @@
 <tourDoneButton "Done">
 <userCountContainer "There are <span class=\"bold\">{{installation_count}}</span> people trying {{title}} right now!">
 <highlightPrivacy "Your privacy">
+
+<noScriptHeading "Uh oh...">
```

Comment:
Unpopular counter proposal: Remove l20n support for the noscript page and just serve all edge-case-users English "Test Pilot requires JavaScript" message.